README.md

```diff
@@ -1,6 +1,19 @@
 # http-request-signature
 An HTTP signature parser and generator fully compliant with the proposed "Signing HTTP Messages" [Internet Draft](https://www.ietf.org/id/draft-cavage-http-signatures-07.txt).
 
+## Installation
```

Comment:
Please make the spacing of all headers consistent in this file (there should be a blank line both above and below each heading).

Reply:
👍

Reply:
The `## Release` and `## License` sections need to be changed as well.

Reply:
(and once you rebase this branch following the merge of #12, so will the `## Status` section.)